CodeMaid/UI/Dialogs/Options/Reorganizing/ReorganizingGeneralViewModel.cs

```diff
@@ -96,6 +97,18 @@ public bool RunAtStartOfCleanup
             set { SetPropertyValue(value); }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether [revert order by access level].
+        /// </summary>
+        /// <value>
+        /// <c>true</c> if [revert order by access level]; otherwise, <c>false</c>.
+        /// </value>
+        public bool RevertOrderByAccessLevel
```

Comment:
Will you please move this up to continue the alphabetical ordering.  It would also be helpful to match the comment style to the others vs. auto-generated.

Perhaps a clearer name for this property would be ReverseAccessLevelOrdering